backend/private-graph/graph/schema.graphqls

```diff
@@ -1281,6 +1281,7 @@ type Query {
 	): ErrorsHistogram!
 	error_group(secure_id: String!): ErrorGroup
 	error_object(id: ID!): ErrorObject
+	error_object_for_log(log_cursor: String!): ErrorObject
```

Comment:
Trying to figure this out over slack but how would one use this endpoint? We can't use it from the logs page since that would be an n+1 query (and request).
I'm hoping we can have some type of error id on the logs table.
Let me know if you had something else in mind!

Reply:
updated in slack - https://highlightcorp.slack.com/archives/C04NUD29BT3/p1678203728066969
enpoint works by having frontend redirect to the existing `/errors/<group>/instances/<instance>` url